doc/modules/model_evaluation.rst

```diff
@@ -6,6 +6,112 @@
+Typical examples are the mean (expected value), the median or a quantile of the
+response variable :math:`Y` (conditionally on :math:`X`).
+
+Once that is settled, use a **strictly consistent** scoring function for that
+target functional, see [Gneiting2009]_.
+For classification **stricly proper scoring rules**, see [Gneiting2007]_, conincide
+with strictly consistent scoring functions.
+See the table further below for examples.
+One could say that consistent scoring functions act as *truth serum* in that
+they guarantee "that truth telling [. . .] is an optimal strategy in
+expectation" [Gneiting2014]_.
+
+Note that for regressors, the prediction is done with :term:`predict` while for
+classifiers it is usually :term:`predict_proba`.
+
+**Decision Making:**
+The most common decisions are done on binary classification tasks, where the result of
+:term:`predict_proba` is turned into a single outcome, e.g., from the predicted
+probability of rain a decision is made on how to act (whether to take mitigating
+measures like an umbrella or not).
+For classifiers, this is what :term:`predict` returns.
+See also :ref:`TunedThresholdClassifierCV`.
+There are many scoring functions which measure different aspects of such a
+decision, most of them are covered with or derived from the :func:`confusion_matrix`.
+
+**List of strictly consistent scoring functions:**
+Here, we list some of the most well known statistical functionals and corresponding
+stritcly consistent scoring functions. Note that the list is not complete and that
+there are more of them.
+For further criteria on how to select a specific one, see [Fissler2022]_.
+
+==================  ===============================================  ================  ==============================
+functional          scoring or loss function                         response y        prediction
+==================  ===============================================  ================  ==============================
+**Classification**
+mean                :ref:`Brier score <brier_score_loss>`            multi-class       ``predict_proba``
+mean                :ref:`log loss <log_loss>`                       multi-class       ``predict_proba``
+mode                :ref:`zero-one loss <zero_one_loss>` :sup:`1`    multi-class       ``predict``, categorical
+**Regression**
+mean                :ref:`squared error <mean_squared_error>`        all reals         ``predict``, all reals
+mean                :ref:`Poisson deviance <mean_tweedie_deviance>`  non-negative      ``predict``, strictly positive
+mean                :ref:`Gamma deviance <mean_tweedie_deviance>`    stricly positive  ``predict``, strictly positive
+median              :ref:`absolute error <mean_absolute_error>`      all reals         ``predict``, all reals
+quantile            :ref:`pinball loss <pinball_loss>`               all reals         ``predict``, all reals
+mode                does not exist
```

Comment:
The message here sounds a bit ambiguous to me. Does not exist at all or is it not implemented in scikit-learn?

Reply:
Does not exist at all.